app/src/main/res/values/rooms.xml

```diff
@@ -7,6 +7,13 @@
         <item>pa5_1</item>
     </string-array>
 
+    <string-array name="floors_pretty">
```

Comment:
can just use the strings associated with the floor ids as they will no longer be used